app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -40,9 +37,6 @@ def track_logout_event
 
   def saml_response_signature_options
     endpoint = SamlEndpoint.new(request)
-    {
-      x509_certificate: endpoint.x509_certificate,
-      secret_key: endpoint.secret_key,
-    }
+    { x509_certificate: endpoint.x509_certificate, secret_key: endpoint.secret_key }
```

Comment:
I prefer the original

Reply:
This is a good example of what I was trying to describe in the sync yesterday re: Prettier's behavior with [print width](https://prettier.io/docs/en/options.html#print-width) maximally filling the target line length. Ours is currently 100 characters, but if this were reduced down to 80 characters, it would break like it was before.